src/couch_replicator_scheduler.erl

```diff
@@ -121,7 +156,8 @@ find_jobs_by_doc(DbName, DocId) ->
 %% gen_server functions
 
 init(_) ->
-    ?MODULE = ets:new(?MODULE, [named_table, {keypos, #job.id}]),
+    EtsOpts = [named_table, {read_concurrency, true}, {keypos, #job.id}],
```

Comment:
The logic there is that for each _scheduler/docs request where we fold through the docs ets table, for each document we'd got to the scheduler and fetch a bit of summary about the state.  If scheduler is rescheduling we don't want the docs request to lock the table fetch a job, then release it.